example/custom_snapshot_resolver_files/BUILD.bazel

```diff
@@ -0,0 +1,21 @@
+load("@aspect_rules_jest//jest:defs.bzl", "jest_test")
```

Comment:
I'm concerned that ppl may not find the right example just based on the folder names. WDYT about adding some README content to the examples maybe? I've just used the filedoc in the BUILD file for simple examples like this

```
"""Shows how to write a custom Jest resolver which writes snapshots of individual files.

Requires that the snapshotted files already exist.
"""
```
or they could have real README files, or maybe a examples/README.md that lists them out?

Reply:
✅